fabric/testdata/feature/feature-svc-whitelist.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: zalando.org/v1
+kind: FabricGateway
+metadata:
+  name: feature-service-whitelist
+  namespace: default
+spec:
+  x-fabric-service:
+  - host: feature-service-whitelist.example.org
+    serviceName: simple-svc
+    servicePort: http
+  x-fabric-admins:
```

Comment:
It would be nice to have the same test case, but without admins. It might catch the bugs around `service-list` being completely empty. (which are not currently present in code).

Reply:
done